common.sh

```diff
@@ -52,8 +51,8 @@ get_svn_revision() {
 
 build_afl() {
   $CC $CFLAGS -c -w $AFL_SRC/llvm_mode/afl-llvm-rt.o.c
-  $CXX $CXXFLAGS -std=c++11 -O2 -c $LIBFUZZER_SRC/afl/*.cpp -I$LIBFUZZER_SRC
-  ar r $LIB_FUZZING_ENGINE *.o
+  $CXX $CXXFLAGS -std=c++11 -O2 -c ${LIBFUZZER_SRC}/afl/afl_driver.cpp -I$LIBFUZZER_SRC
+  ar r $LIB_FUZZING_ENGINE afl_driver.o afl-llvm-rt.o.o
```

Comment:
Not a typo, but it may not the most straightforward name either, just the default naming convention for `clang -c`. Would renaming this be preferable?

The code primarily comes from https://github.com/llvm-mirror/llvm/blob/master/lib/Fuzzer/afl/afl_driver.cpp#L29